_studio/mfx_lib/shared/include/mfx_h264_encode_struct_vaapi.h

```diff
@@ -989,6 +989,16 @@ enum
     ENCODE_QUERY_STATUS_ID                  = 0x121
 };
 
+// Slice structure
+enum
```

Comment:
Do you want to say that H.265 reuses H.264 header file(s)?! I always thought this is not the case due to significant code duplication....

Reply:
yes

Reply:
oh, my...